src/main/java/com/jfrog/ide/idea/scan/data/applications/JFrogApplicationsConfig.java

```diff
@@ -22,15 +23,16 @@ public class JFrogApplicationsConfig {
 
     public static JFrogApplicationsConfig createApplicationConfigWithDefaultModule(Project project) {
         JFrogApplicationsConfig applicationsConfig = new JFrogApplicationsConfig();
-
-        ModuleConfig defualtModuleConfig = new ModuleConfig();
-        defualtModuleConfig.setSourceRoot(getProjectBasePath(project).toAbsolutePath().toString());
-        defualtModuleConfig.setExcludePatterns(convertToSkippedFolders(GlobalSettings.getInstance().getServerConfig().getExcludedPaths()));
-
+        Set<Path> paths = com.jfrog.ide.idea.scan.ScanUtils.createScanPaths(project);
         applicationsConfig.modules = new ArrayList<>();
-        applicationsConfig.modules.add(defualtModuleConfig);
+
+        for (Path path : paths) {
```

Comment:
Let's use Utils.consolidatePaths to eliminate conjunctions between paths:

```suggestion
        for (Path path : Utils.consolidatePaths(paths)) {
```